### ISSUE TITLE
not accepting initialTags, receiving undefined object

### ISSUE THREAD

**Issue by socialcode-rob1** [peterp/react-native-tags#9]
Using latest v1.0.0 after #7 fix

I'm receiving this ...
<img width="270" alt="screen shot 2018-04-11 at 11 19 09 pm" src="https://user-images.githubusercontent.com/540616/38654743-dc847604-3dde-11e8-80bd-e10ac0df2569.png">

When trying to assign an array to initialTags
```          
<Tags  
  initialText=""  
  initialTags={data.keywords}   
  onChangeTags={tags => console.log(tags)}  
  onTagPress={(index, tagLabel, event) => console.log(index, tagLabel, event)}  
  inputStyle={{ backgroundColor: 'white' }}  
/>  
```

**Note** I was using the same initialTags prop in the previous version without any issue.  The only thing updated (to match example doc) was the onChangeTags. #

**Comment by peterp** [comment]
@socialcode-rob1 which version of react and react-native are you using?

**Comment by socialcode-rob1** [comment]
I'm using ...  

react: v16.2  
react-native: v0.53.3

**Comment by peterp** [comment]
My bad, I'll fix this in a few...

**Comment by socialcode-rob1** [comment]
I upgraded my react and everything works now.

**Comment by peterp** [comment]
Ok, thanks, I still think it's worth it to make it backwards compatible.

**Comment by aryankranthi** [comment]
@peterp  can you make it backwards compatible with older versions like 16.0.0

**Comment by peterp** [comment]
@aryankranthi absolutely, sorry about that.

**Comment by peterp** [comment]
Closed in 1.2.1.
